packages/elements-core/src/components/Docs/Model/Model.tsx

```diff
@@ -6,14 +7,22 @@ import * as React from 'react';
 
 import { SchemaAndDescription } from '../../SchemaAndDescription';
 import { DocsComponentProps } from '..';
+import { InternalBadge } from '../HttpOperation/Badges';
 
 export type ModelProps = DocsComponentProps<JSONSchema7>;
 
 const ModelComponent: React.FC<ModelProps> = ({ data, className, headless }) => {
+  const isInternal = !!data['x-internal'];
+
   return (
     <div className={cn('Model MarkdownViewer', className)}>
       {!headless && data.title !== void 0 && <h1 className={Classes.HEADING}>{data.title}</h1>}
 
+      {isInternal && (
+        <HStack spacing={2} mt={3} mb={12}>
+          {isInternal && <InternalBadge />}
```

Comment:
doubled `isInternal` condition here

Reply:
Thank you for flagging! Addressed with 6e25ce5